packages/markdown-cli/test/cli.js

```diff
@@ -148,3 +148,16 @@ describe('markdown-cli (docx)', () => {
         });
     });
 });
+
+describe('markdown-cli (ooxml)', () => {
+    // Partial Acceptance test
+    const inputOOXMLFile = path.resolve(__dirname, 'data/acceptance', 'partial_acceptance.xml');
+
+    describe('#parse', () => {
+        it('should roundtrip ciceromark_parsed <-> ooxml', async () => {
+            const result = await Commands.transform(inputOOXMLFile, 'ooxml', [], 'ciceromark_parsed', null, {}, { roundtrip: true });
+            result.should.equal(fs.readFileSync(inputOOXMLFile, 'utf-8'));
+        });
```

Comment:
I think you would also need to check for going from cicero mark to OOXML. And for that, you would need to create a JSON file partial_acceptance.json.

Reply:
@algomaster99 Will update the filenames and tests ASAP.

Reply:
> I think you would also need to check for going from cicero mark to OOXML. And for that, you would need to create a JSON file partial_acceptance.json.

@algomaster99 
Regarding this, we already check for the conversion test so why do we do this one. `roundtrip: true` also pass so I am not able to understand the need to add this.
In `transform.js`, we have already done the test for both.

Reply:
> Regarding this, we already check for the conversion test so why do we do this one. roundtrip: true also pass so I am not able to understand the need to add this.

We aren't checking for equality for ciceromark_parsed in between so that's why it's needed.

> In transform.js, we have already done the test for both.

Yeah, I am aware. I even pointed this out in the meeting. With that logic, I think we can completely remove any testing related to this PR from `markdown-cli`.